src/Tribe/Capabilities.php

```diff
@@ -1,9 +1,41 @@
 <?php
 
 class Tribe__Events__Capabilities {
-	public $set_initial_caps = false;
-	private $cap_aliases = array(
-		'editor' => array( // full permissions to a post type
+
+	/**
+	 * The transient key for delayed capabilities updates.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	protected $key_delayed_capabilities_update = '_tribe_events_delayed_capabilities_update';
+
+	/**
+	 * An array of roles to update capabilities.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	protected $roles = [
+		'administrator',
+		'editor',
+		'author',
+		'contributor',
+		'subscriber',
+	];
+
+	/**
+	 * An array of capabilities aliases by role.
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	private $cap_aliases = [
+		//Full permissions to a post type.
```

Comment:
:no_entry_sign: No space found before comment text; expected "// Full permissions to a post type." but found "//Full permissions to a post type."

<sub>hash: 16ae6305ad98e3dbafe0fa1a09fc7679</sub>